CONTRIBUTING.md

```diff
@@ -119,7 +119,7 @@ nx test jest
 
 ### Running E2E Tests
 
-**Use Node 16 and NPM 8. E2E tests won't work on Node 15 and NPM 7.**
+**Use Node 18 and NPM 8. E2E tests won't work on Node 15 and NPM 7.**
```

Comment:
Should we say that it also does not work with 16? The documentation does not work with 16. Do e2e tests work with 16?